### PR TITLE
Add support for Azure DevOps urls

### PR DESCRIPTION
**Pull request by sbonebrake**

### CODE REVIEW

**Comment by sbonebrake** [comment]
Tests work locally. appveyor is failing with: "Error: Cannot find NPM_TOKEN". Not sure how that was set up.

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "git-urls",
-  "version": "0.2.5",
+  "version": "0.2.6",
```
Comment (qinezh) [inline package-lock.json:3]: Can you please keep the version unchanged in this PR? thanks

**Comment by qinezh** [comment]
> Tests work locally. appveyor is failing with: "Error: Cannot find NPM_TOKEN". Not sure how that was set up.

Never mind, the CI issue has already been fixed.

File: package-lock.json (hunk 1 above)
Comment (qinezh) [reply package-lock.json:3]: Oh I can handle it myself.

**Comment by qinezh** [comment]
LGTM 🚀

**Comment by sbonebrake** [comment]
Whoops. Didn't realize I updated the version. Thanks for merging!

**Pull request merged**